stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/ChannelActivity.kt

```diff
@@ -202,6 +203,7 @@ class ChannelActivity : AppCompatActivity() {
                         .wrapContentHeight()
                         .align(Alignment.Center),
                     shape = RoundedCornerShape(16.dp),
+                    isMuted = listViewModel.isChannelMuted(selectedChannel!!.cid),
```

Comment:
Let's not do this. I know it's a bit uglier, but I'd rather capture a value and then check that, than have `!!` in code.

Something like:

```
val currentSelectedChannel = selectedChannel

if (currentSelectedChannel != null) {
 // ...
}
```

The state delegate isn't the best in this case, but I think this is okay too.

Reply:
Yes, works for me. Implemented like that at fist, but this produces IDE warning. Done, captured the value in a temp `currentSelectedChannel` value.